### ISSUE TITLE
VSCodeInsider_DefaultSetting_ScreenReader:VoiceOver does not announces label name for the edit filed.

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/vscode#89097]
**Environment Details:​**
VSCode Version: 1.41.0-insider (System setup)
Operating System: MacOS
Screen reader: VoiceOver
Mac version: 10.15.1 (19B88)

**Repro Steps:​**
1. Open VS Code insider 
2. Navigate till "Extension" button and search for the "Visual Studio Online"
3. Click on "Visual Studio Online" and navigate till "Remote Explorer" ,click on it.
4. Navigate till "+" button beside the  "VS ONLINE" control and click on it .
5. Select any existing environment and then again click on the "+" button and navigate till "Default setting" ,click on it.
6. Start the VoiceOver.
7. Observe the screen reader announces label name or not.

**Actual:**
VoiceOver does not announces the label name for the edit filed.

**Expected:​**
VoiceOver should  announces the label name for the edit filed.

**Same issue observed for the below scenario:**
**1.Customer Setting**
1.Open VS Code insider 
2.Navigate till "Extension" button and search for the "Visual Studio Online"
3.Click on "Visual Studio Online" and navigate till "Remote Explorer" ,click on it.
4.Navigate till "+" button beside the  "VS ONLINE" control and click on it .
5.Select any existing environment and then again click on the "+" button and navigate till "Custom setting" ,click on it.
6.Verify that all the elements of "Customer setting" for new environment" pop up are accessible and MAS Compliant

**​User Impact:​**
Screen reader user does not understand the name of the regarding the control.

**Comment by isidorn** [comment]
Dup of https://github.com/microsoft/vscode/issues/88839
